hack/charts/cluster-api-operator/values.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "definitions": {
+    "nameNamespaceVersion": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "namespace": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        }
+      },
+      "required": ["name"],
+      "additionalProperties": false
+    }
+  },
+  "properties": {
+    "core": {
+      "$ref": "#/definitions/nameNamespaceVersion"
+    },
+    "bootstrap": {
+      "type": "array",
+      "minItems": 1,
```

Comment:
Setting `minItems: 1` might break the tests but how useful is this chart if you do not specify all of the providers? Are any of these actually optional?

Reply:
This did indeed break the tests. Removed.